src/maintainer/adding_pkgs.rst

```diff
@@ -110,6 +110,36 @@ Post staging process
 * If you want to make a change to the recipe, send a :term:`PR` to the git repository from a fork. Branches of the main repository are used for maintaining different versions only.
 
 
+.. _feedstock-repository-structure:
+
+Feedstock repository structure
+------------------------------
+
+Once the PR having the recipe for a package is merged in the ``staged-recipes`` repository, a new repository is created automatically called ``<package-name>-feedstock``.
+A feedstock is made up of a conda recipe (the instructions on what and how to build the package) and the necessary configuration files for automatic builds using freely available continuous integration services.
```

Comment:
Let's add "(CI)" after "continuous integration", since we mention CI configuration files later in this section. That will make things more clear.